eng/pipelines/dotnet-framework.yml

```diff
@@ -11,9 +11,8 @@ stages:
 - template: ../common/templates/stages/build-test-publish-repo.yml
   parameters:
     ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-      windowsAmdBuildJobTimeout: 300
-      windowsAmdTestJobTimeout: 120
+      windowsAmdBuildJobTimeout: 150
     ${{ if eq(variables['System.TeamProject'], 'public') }}:
       buildMatrixType: platformVersionedOs
       customBuildLegGrouping: pr-build
-      windowsAmdBuildJobTimeout: 420
+      windowsAmdBuildJobTimeout: 300
```

Comment:
Considering that the build for this PR had a leg that took 4:50, I think this should at least be changed to 330.

Reply:
Changed as suggested